SELF_ASSESSMENT.md

```diff
@@ -144,6 +144,42 @@ vulnerabilities, etc.--->
 
 ### Communication Channels
 
+#### Internal
```

Comment:
We use the CNCF Slack workspace for internal communication and we have an internal Oversight Committee meeting every two weeks.